db/seeds/development/7_groups.rb

```diff
@@ -0,0 +1,18 @@
+@alphabet = ("a".."z").to_a
+
+def generate_group(conference_id)
+  new_group = Group.create(
+    name: @alphabet.shift,
+    conference_id: conference_id,
+  )
+end
+
+(1..5).each do
+  generate_group(1)
+  puts "Created group for Conference 1"
```

Comment:
Don't forget a period at the end, and can use single quotes since there's no string interpolation here!